### PR TITLE
Add heroku Button

### PR DESCRIPTION
**Pull request by friism**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET vNext Sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
+    "env": {
+	"BUILDPACK_URL": "https://github.com/friism/heroku-buildpack-mono#aspnet-vnext"
```
Comment (Eilon) [inline app.json]: Hi there! Do you know if there's an official Heroku build thingy (a technical term!) for ASP.NET 5? We generally prefer not to link to personal accounts due to support and reliability issues.

**Comment by Eilon** [comment]
And regarding this change:

`src/MusicStore/Views/_ViewStart.cshtml → src/MusicStore/Views/_viewstart.cshtml `

The correct casing is `_ViewStart.cshtml`. It may have been broken temporarily in some builds, but going forward the mixed case one is the correct one.

File: app.json (hunk 1 above)
Comment (friism) [inline app.json]: Let me get back to you on that (I work at Heroku but am currently working on this on my own).

Comment (Eilon) [inline app.json]: Yeah I kind of figured that from your profile :smile:

**Comment by Praburaj** [comment]
@friism - Thanks for the PR. And could you also send the PR to the `dev` branch?

**Comment by friism** [comment]
@Eilon I factored out the casing change and added a note detailing how it might break with non-nightly MVC versions.

**Comment by friism** [comment]
I'm working on the CSA

File: app.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET vNext Sample app",
```
Comment (Eilon) [inline app.json]: BTW should say `ASP.NET 5`

**Comment by Eilon** [comment]
Cool just let us know when you've signed it and I can verify on our side.

**Comment by friism** [comment]
Closing in favor of https://github.com/aspnet/MusicStore/pull/411

**Pull request closed without merging**